### PR TITLE
MODE-2518: Attach binary content jcr:content

### PR DESCRIPTION
**Pull request by okulikov**
This PR fixes bug when property with null value (ie just created node) is not displayed by property editor.

### CODE REVIEW

File: web/modeshape-web-explorer/src/main/java/org/modeshape/web/server/JcrServiceImpl.java [hunk 1]
```diff
@@ -352,15 +352,16 @@ private AccessControlList findAccessList( AccessControlManager acm,
...
             
-            Property p;
+            Property p = null;
             try {
                 p = node.getProperty(def.getName());
-                String display = values(def, p);
-                String value = p.isMultiple() ? multiValue(p)
-                        : singleValue(p, def, repository, workspace, path);
-                list.add(new JcrProperty(name, type, value, display));
             } catch (PathNotFoundException e) {
```
Comment (hchiorean) [inline web/modeshape-web-explorer/src/main/java/org/modeshape/web/server/JcrServiceImpl.java]: should this exception be ignored ?

Comment (okulikov) [inline web/modeshape-web-explorer/src/main/java/org/modeshape/web/server/JcrServiceImpl.java]: yes. it means that property is not assign yet.

**Comment by hchiorean** [comment]
@rhauch are you ok with merging this ?

**Comment by rhauch** [comment]
@hchiorean, looks good.

**Pull request merged**